manifests/hammer/plugin.pp

```diff
@@ -0,0 +1,24 @@
+# Configuration for hammer development
+class katello_devel::hammer::plugin (
+  $config_template = undef,
```

Comment:
For scoping I think it'd make more sense to actually pass in the content rather than the template.